steps/sms-notify/sms-confirmation/template.html

```diff
@@ -17,7 +17,13 @@
     {{ header(content.title, size="large") }}
     <p>{{ content.mobileNumber }}
         <span class="bold-small">
-            {{ session.EnterMobile_mobileNumber if (session.EnterMobile_mobileNumber) and session.SendToNumber_useSameNumber == 'no' else session.AppellantDetails_phoneNumber }}
+
+            {% if (session.AppellantDetails_phoneNumber | isMobilePhone) and (session.SendToNumber_useSameNumber === 'yes') %}
```

Comment:
Ben, would it be useful to move this logic into a model and keep the view focused on replacing placeholders? That could help in having unit tests "guarding" the logic. I hope it makes sense.

Reply:
@KarolJastrzebski you're right, there's way too much logic in the template. Unfortunately the framework is not mature enough atm. We shouldn't be modifying the session either. I've spoken to @michaeldfallen and he has mentioned that "Ref fields and adding getter to locals" is next on his list which will allow @bendiggle to implement this the preferred way. Ben will close this PR shortly.